src/actions/types.js

```diff
@@ -0,0 +1,2 @@
+export const GET_ERRORS = 'GET_ERRORS';
```

Comment:
actions should be inside components folder

Reply:
what does it mean? that every component need  to have his own action?

Reply:
yes :)